api/src/pcapi/core/mails/transactional/sendinblue_template_ids.py

```diff
@@ -106,10 +106,10 @@ class TransactionalEmail(Enum):
     # PRO EMAIL
 
     BOOKING_CANCELLATION_BY_BENEFICIARY_TO_PRO = models.TemplatePro(
-        id_prod=609, id_not_prod=51, tags=["pro_annulation_offre"]
```

Comment:
Attention, je vois les paramètres envoyés pour l'email **379**, qui correspondent bien à ceux dans le _template_ Brevo : 
https://github.com/pass-culture/pass-culture-main/blob/v290.0.3/api/tests/core/mails/transactional/bookings/booking_cancellation_by_beneficiary_to_pro_test.py#L79

Mais en regardant le _template_ **379** dans Brevo, je vois que sont notamment utilisés : 
```
params.NOM_OFFRE
params.NOM_LIEU
params.PRIX
params.QUANTITE
params.HEURE
```
**Ces paramètres ne sont donc pas remplis par le backend !**
La correction se fera probablement dans Brevo par souci d'homogénéité, car on utilise partout des noms de variables en anglais.
Mais il est préférable de s'assurer que les paramètres « tombent en face » avant _merge_.

Tu peux voir le problème avec le _template_ **163** sur le compte de test Brevo.
En activant l'envoi d'email depuis l'environnement de développement en local, ça se teste bien :slightly_smiling_face:

Reply:
ça devrait être bon maintenant